README.md

```diff
@@ -1,2 +1,51 @@
-# cli
-[Experimental] New Apollo CLI and Language Server written in Rust
+# [Experimental] Apollo CLI
+
+
+This project is an early experiment at replacing the functionality of the current apollo-tooling repo with a new, and much better, CLI / LSP experience. It is being written in rust to have maximum portability and is aimed at a new command structure, config / onboarding experience, etc. Its greenfield ;)
+
+## Contributing
+
+This is very early days for this project. It is setup as a multi-crate rust project. To get started you need to have the rust toolchain installed. The best way to do this is to visit [the amazing rust site](https://www.rust-lang.org/learn/get-started) and follow their install instructions.
+
+To ensure you are ready, you should be able to run this:
+
+```sh
+cargo --version
+```
+
+and get something back that looks like this:
+
+```sh
+cargo 1.42.0 (86334295e 2020-01-31)
+```
+
+### Running the project
+To run the project locally, you can use `cargo` to build and exexecute the bin of the `apollo` crate.
+
+```
+cargo run --bin apollo
```

Comment:
Let's put `cargo run` right up top, akin to the `npm install && npm run` part of so many JS readmes.

Also, if we get rid of the `graphql` crate—or make it a library crate, I think?—we won't have to specify the `--bin apollo`, we can just do `cargo run`

Reply:
@queerviolet we can get rid of the `--bin apollo` but I suspect we will want another crate pretty quickly into this build out. I'll add a section about running the CLI as well. I suspect there is a way to make only one of the packages a "bin" command to ease this as well

Reply:
@queerviolet yep! That worked great :)